src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -294,35 +298,33 @@ private static void updateMapUrlAndSizeLabel(final DownloadFileDescription map,
   }
 
   private static String createLabelText(final MapAction action, final DownloadFileDescription map) {
-    final String DOUBLE_SPACE = "&nbsp;&nbsp;";
+    final String doubleSpace = "&nbsp;&nbsp;";
 
-    final long mapSize;
-    String labelText = "<html>" + map.getMapName() + DOUBLE_SPACE + " v" + map.getVersion();
+    final StringBuilder sb = new StringBuilder();
```

Comment:
True, but I'd split up this line into multiple append function calls e.g. `builder.append("<html>").append(map.getMapName())....` to be consistent